### ISSUE TITLE
Additional prefix flexibility

### ISSUE THREAD

**Issue by danciupuliga** [johnotander/rework-class-prefix#7]
Hello,

Great plugin!

I am currently attempting to use it to prefix a CSS library from `.libraryName-className` to `.libraryName-version-className`. The current functionality is pretty specific to allowing a prefix only after the `.` of the classname as expected based on the name. 

I would like to open a PR to allow for a more generic approach allowing the user to pass both the prefix and the `split()` parameter, but wanted to see if maybe you consider that type of functionality out of scope for this plugin.

Thanks
